helmfile/charts/networkpolicy-service/templates/certmanager-np.yaml

```diff
@@ -24,23 +24,32 @@ spec:
     - Ingress
     - Egress
   ingress:    
-      # - from:
-      #   {{- range $IP := .Values.certmanager.letsencrypt.ips }}
-      #   - ipBlock:
-      #       cidr: {{ $IP }}
-      #   {{- end }}
-      #   ports:
-      #     - protocol: TCP
-      #       port: 443      
-    - ports:      
-      - protocol: TCP
-        port: 9402 # promtheus metrcis            
+    - from:
+      {{- range $IP := .Values.certManager.letsencrypt.ips }}
+      - ipBlock:
+          cidr: {{ $IP }}
+      {{- end }}
```

Comment:
Ignore my previous comment about letsencrypt ips not being used, I was looking at an old commit.

Shouldn't this be on the ingress? Letsencrypt should never be able to talk to certmanager without going through the ingress, right?

Reply:
I allowed everything on 443 in ingress-nginx chart

Reply:
as ingress rule

Reply:
Do you want me to specifically allow this at 443 ?

Reply:
Ok. Yeah for nginx I think it makes sense to have 443 and 80 open or only allow traffic from a Loadbalancer if you want you can add that as an option.

But then you should be able to remove this ingress rule from cert-manager. But you can use the letsencrypt ips to configure an egress rule from cert-manager to letsencrypt